signing/secp256k1/impl/src/main/java/tech/pegasys/signers/secp256k1/multikey/SigningMetadataTomlConfigLoader.java

```diff
@@ -50,15 +50,15 @@ public SigningMetadataTomlConfigLoader(final Path rootDirectory) {
     this.tomlConfigsDirectory = rootDirectory;
   }
 
-  Optional<SigningMetadataFile> loadMetadataForAddress(final String address) {
+  Optional<SigningMetadataFile> loadMetadata(final String identifier) {
```

Comment:
Why the rename if this is still specific to Ethereum addresses?

Reply:
its not longer specific - this gets used for Addresses and public keys - its now just a filename

Reply:
The normalizeAddress is calls below through me off. Do we still need to do the normalizeAddress in this function?